### PR TITLE
Altered Registration validations

### PR DESCRIPTION
**Pull request by ming-0630**

### CODE REVIEW

File: src/main/java/com/oodj/vaccspace/controllers/register/RegisterController.java [hunk 1]
```diff
@@ -55,13 +55,56 @@ void onBackToLoginPressed(ActionEvent event) {
...
+        }
+
+        //Identification Number Validation
+        if (!vm.getIdentificationNumber().matches("^[0-9]*$")) {
+            Page.showDialog(container.getScene().getWindow(), DialogType.ERROR, "Error: Invalid IC Number", "Identification number must only have numbers!");
+            return;
+        }
+
+        //Email Validation
+        if (!vm.getEmail().matches("^[A-Za-z0-9+_.-]+@(.+)$")) {
```
Comment (wooneusean) [inline src/main/java/com/oodj/vaccspace/controllers/register/RegisterController.java]: This regex matches `xyz@mail`, which is kinda wrong, please make it match `xyz@mail.com`

**Review by wooneusean — CHANGES_REQUESTED** [review]
Email regex inaccurate, besides that all good.

File: src/main/java/com/oodj/vaccspace/controllers/register/RegisterController.java (hunk 1 above)
Comment (ming-0630) [reply src/main/java/com/oodj/vaccspace/controllers/register/RegisterController.java]: Changes have been made, not sure if the PR will update themselves but yea lmk if i need to create another PR

Comment (wooneusean) [reply src/main/java/com/oodj/vaccspace/controllers/register/RegisterController.java]: looks good to me

**Review by wooneusean — APPROVED** [review]

**Pull request merged**